packages/web-components/src/select/select.spec.md

```diff
@@ -0,0 +1,100 @@
+# Select
+
+## Component Description
+
+The browser-native select control component allows users to choose one option from a dropdown list. It is commonly used in forms for selecting values from a predefined set of options.
```

Comment:
I'd prefer that we not use the term "dropdown" here as it could create confusion around why we have "dropdown" as well. Additionally, it would probably make sense to elaborate on why we have three components here and specify which one this is, etc.